src/value.rs

```diff
@@ -181,8 +181,8 @@ impl From<SentryString> for Value {
     }
 }
 
-impl From<&String> for Value {
-    fn from(value: &String) -> Self {
+impl From<String> for Value {
+    fn from(value: String) -> Self {
         SentryString::new(value).into()
     }
 }
```

Comment:
The reason this was a `&String` was because it doesn't get consumed.

Reply:
I kinda noticed I was using `AsRef<str>` for `SentryString`, I removed it and made it consistent with not consuming too.
What do you guys think is better?